packages/docs-theme/src/components/interfaceTable.tsx

```diff
@@ -31,79 +30,91 @@ function propTag(intent: Intent, title: string, ...children: React.ReactNode[])
     );
 }
 
-const renderPropRow = (prop: IInheritedPropertyEntry) => {
-    const { documentation, inheritedFrom, name, optional } = prop;
-    const { default: defaultValue, deprecated, internal } = prop.tags;
-
-    if (internal) {
-        return undefined;
+function renderPropType(prop: ITsProperty | ITsMethod) {
+    if (isTsProperty(prop)) {
+        const formattedType = prop.type.replace(/\b(JSX\.)?Element\b/, "JSX.Element");
+        return (
+            <code className="docs-prop-type">
+                <strong>{formattedType}</strong>
+                <em className="docs-prop-default pt-text-muted">{prop.defaultValue}</em>
+            </code>
+        );
+    } else {
+        return (
+            <code className="docs-prop-type">
+                <strong>{prop.signatures[0].type}</strong>
+            </code>
+        );
     }
+}
+
+function renderPropRow(prop: ITsProperty | ITsMethod) {
+    const { flags: { isDeprecated, isExternal, isOptional }, inheritedFrom, name } = prop;
 
     const classes = classNames("docs-prop-name", {
-        "docs-prop-is-deprecated": deprecated != null,
-        "docs-prop-is-internal": internal,
-        "docs-prop-is-required": !optional,
+        "docs-prop-is-deprecated": !!isDeprecated,
```

Comment:
really not a fan of this, especially for existing code, since I want to enable [strict-boolean-expressions](https://palantir.github.io/tslint/rules/strict-boolean-expressions/) in our lint config and this is a step backwards IMO